site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/SupportForm.js

```diff
@@ -0,0 +1,290 @@
+                                required
+                                fluid={false}
+                                width={4}
+                            />
+                            <TextField
+                                disabled={isUserAuthenticated}
+                                label='Email'
+                                fieldPath="email"
+                                required
+                                fluid={false}
+                                width={4}
+                            />
+                            <CategoryDropdown
+                                categories={categories}
+                                defaultCategory={defaultCategory}
+                            />
+                            <TextField
+                                label='Subject'
+                                fieldPath="subject"
+                                required
+                                fluid={false}
+                                className='mt-10'
+                            />
+                            <TextAreaField
+                                label='Description'
+                                fieldPath="description"
+                                required
+                            />
+                            <div className="field">
+                                <label>
+                                    {`Browser & OS`}
+                                </label>
+                                <ToggleField
+                                    fieldPath='sysInfo'
+                                    offLabel='Include browser and system information to assist us with narrowing down the cause of your problem.'
+                                    onLabel='Include browser and system information to assist us with narrowing down the cause of your problem.'
+                                    offValue={false}
+                                    onValue={true}
+                                />
+                                <label className="helptext">
+                                    {sysInfo}
+                                </label>
+                            </div>
+                            <div className="field">
+                                <label>
+                                    {`Files`}
+                                </label>
+                                <FileUploader
+                                    dropzoneParams={dropzoneParams}
+                                    maxFileSize={maxFileSize}
+                                    name='files'
+                                    currentFiles={this.state.files}
+                                    handleDelete={(file) => {
+                                        const newFiles = this.state.files;
+                                        _remove(newFiles, file);
+                                        this.updateFiles(newFiles, setFieldValue)
```

Comment:
File is removed from the component state and Formik. The component state is passed to the table for files listing. Formik is used to create the request object to post the form.

Note that `react-dropzone` is not being updated, I did not find a way of doing it.